pdns/dnsdistdist/configure.ac

```diff
@@ -56,6 +54,8 @@ AS_IF([test "x$enable_hardening" != "xno"], [
   AC_LD_RELRO
 ])
 
+PDNS_CHECK_OS
+
 PDNS_ENABLE_SANITIZERS
 
 PDNS_CHECK_PANDOC
```

Comment:
No PDNS_CHECK_CLOCK_GETTIME? And it looks like pdns_check_clock_gettime.m4 is not symlinked to in dnsdist's m4 folder.